flask_gtts.read() fails: read.html not found

```log
tests/templates/6175.html:1: in top-level template code
    {{read()}}
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <flask_gtts.main.gtts object at 0x7fc3eeb0e790>, id = '.toRead', mouseover = False

    def read(self, id='.toRead', mouseover=False):
        '''Read an HTML element inner text.
    
        Parameters
        ----------
        id : str, optional
            HTML element css selector, by default '.toRead'
        mouseover : bool, optional
            Read text on `mouseover` event instead of `click`, by default False
    
        Returns
        -------
        str
            Safe JavaScript to read an HTML element content.
        '''
        if not self.route:
            try:
                self.set_route()
            except Exception:
                pass
    
        file_path = os.path.join(os.path.dirname(__file__), 'read.html')
    
>       with open(file_path) as html_file:
E       FileNotFoundError: [Errno 2] No such file or directory: '/home/mrf3/Projects/flask_gtts/flask_gtts/read.html'

flask_gtts/main.py:137: FileNotFoundError
```

--- a/flask_gtts/about.py
+++ b/flask_gtts/about.py
@@ -1,4 +1,4 @@
-__version__ = '0.15'
+__version__ = '0.16'
 __doc__ = 'A Flask extension to help in generating Google Text-To-Speech files.'
 __license__ = 'MIT'
 __author__ = 'Mohamed Feddad'

--- a/setup.py
+++ b/setup.py
@@ -39,10 +39,10 @@ setup(
     long_description_content_type='text/markdown',
     packages=['flask_gtts'],
     zip_safe=False,
-    include_package_data=True,
     platforms='any',
     install_requires=requirements,
     setup_requires=test_requirements,
+    package_data={'flask_gtts': ['read.html']},
     keywords=['flask', 'extension', 'google', 'text', 'speech',
               'gTTS', 'TTS', 'text-to-speech'],
     classifiers=[

